### PR TITLE
[Related #88] Player list component

### PR DESCRIPTION
**Pull request by supunTE**
- Player list item component
- Storybook component

### CODE REVIEW

File: apps/client/src/pages/room/Room.page.tsx [hunk 1]
```diff
@@ -10,6 +10,15 @@ export function Room(): ReactElement {
...
       </Text>
+      {/*TODO: Implement PlayerList*/}
+      {/*<div className={cs('my-12')}>*/}
+      {/*  <ListItem*/}
+      {/*    title='JohnDoe'*/}
+      {/*    imgURL='https://avatars.dicebear.com/api/open-peeps/737373.svg'*/}
+      {/*    number={1}*/}
+      {/*    rightText='64 wpm'*/}
+      {/*  />*/}
+      {/*</div>*/}
```
Comment (dalanad) [inline apps/client/src/pages/room/Room.page.tsx:21]: Was this commented part intended to be commented ?

**Review by dalanad — COMMENTED** [review]
Screenshot 2023-06-30 at 09 06 32
@supunTE Text seems unaligned

**Review by dalanad — CHANGES_REQUESTED** [review]
Fix the alignment make the text center vertically

File: apps/client/src/pages/room/Room.page.tsx (hunk 1 above)
Comment (supunTE) [reply apps/client/src/pages/room/Room.page.tsx:21]: This page is entirely recreated on PR https://github.com/the-ai-team/razor/pull/95

**Comment by supunTE** [comment]
The alignment was already in place, but due to variations in the top padding caused by the image, it might appear misaligned. To address this, I've added a background color, which effectively eliminates the perception of misalignment.

image

**Review by dalanad — APPROVED** [review]

**Pull request merged**